chef/cookbooks/hyperv/recipes/config.rb

```diff
@@ -100,9 +112,10 @@
             :vlan_start => vlan_start,
             :vlan_end => vlan_end
            )
+  notifies :restart, "service[neutron-hyperv-agent]", :immediate
 end
 
-cookbook_file "#{node[:openstack][:config]}\\interfaces.template" do
+cookbook_file "#{node[:openstack][:config]}/interfaces.template" do
```

Comment:
no,  but it doesn't hurt and is consistent with the other changes. Windows supports both path separators.